xml/System.Globalization/CultureInfo.xml

```diff
@@ -1173,19 +1173,22 @@ If `name` is <xref:System.String.Empty?displayProperty=nameWithType>, the constr
       </ReturnValue>
       <Docs>
         <summary>Gets or sets the <see cref="T:System.Globalization.CultureInfo" /> object that represents the culture used by the current thread.</summary>
-        <value>An object that represents the culture used by the current thread.</value>
+        <value>The culture used by the current thread.</value>
         <remarks>
           <format type="text/markdown"><![CDATA[
 
 ## Remarks
- The <xref:System.Globalization.CultureInfo> object that is returned by this property and its associated objects determine the default format for dates, times, numbers, currency values, the sorting order of text, casing conventions, and string comparisons.
 
- The current culture is a property of the executing thread. Retrieving the value of the <xref:System.Globalization.CultureInfo.CurrentCulture%2A?displayProperty=nameWithType> property is a more performant equivalent of retrieving the <xref:System.Globalization.CultureInfo> object returned by the `Thread.CurrentThread.CurrentCulture` property.
+The <xref:System.Globalization.CultureInfo> object that's returned by this property and its associated objects determine the default format for dates, times, numbers, currency values, the sorting order of text, casing conventions, and string comparisons.
+
+The current culture is a property of the executing thread. Retrieving the value of the <xref:System.Globalization.CultureInfo.CurrentCulture%2A?displayProperty=nameWithType> property is a more performant equivalent of retrieving the <xref:System.Globalization.CultureInfo> object returned by the `Thread.CurrentThread.CurrentCulture` property. When you set this property to a <xref:System.Globalization.CultureInfo> object that represents a new culture, the value of the `Thread.CurrentThread.CurrentCulture` property also changes.
```

Comment:
I hope we can mention something that `Thread.CurrentThread.CurrentCulture` kind of obsolete and CultureInfo.CurrentCulture should be used instead all the time.